src/api-docs.js

```diff
@@ -282,6 +282,90 @@ const defaultApiDocs = {
         },
       },
     },
+    '/fireblocks/start': {
+      post: {
+        operationId: 'fireblocksWalletStart',
+        summary: 'Start a fireblocks client wallet on Hathor network.',
+        requestBody: {
+          description: 'Data to start the wallet',
+          required: true,
+          content: {
+            'application/json': {
+              schema: {
+                type: 'object',
+                required: ['wallet-id', 'xpub-id'],
+                properties: {
+                  'wallet-id': {
+                    type: 'string',
+                    description: 'Define the key of the corresponding wallet it will be executed the request.'
+                  },
+                  'xpub-id': {
+                    type: 'string',
+                    description: 'Key name of the desired xPub on the xpub map config.'
+                  },
```

Comment:
I can change this, but I think the main difference here is that the xpub config is not special to the fireblocks (unlike hsm-key) and can be useful for the main start api, for instance we can have read-only wallets started from config.

We can also change to `xpub` now and later if we add the xpub config on the main start api we can also change it here.
What do you think?

Reply:
I like this approach!

That way this will be a more integrated feature, rather than feeling that each external signature integration has its own style.

I mention this because, for example, we spent a lot of effort to make the routes and parameters for the `p2sh` and `tx-proposal` features look similar. I feel this is a similar case of trying to make the api as consistent as possible.